### DISCUSSION TITLE
How to add to the wg0.conf Server Config

### DISCUSSION THREAD

**Discussion by Tyfosho** [wg-easy/wg-easy#143] [Q&A] [answered]
Hi all,

I'm trying to add the following lines to my wg0.conf file:

PostUp = iptables -A FORWARD -i %i -j ACCEPT; iptables -A FORWARD -o %i -j ACCEPT; iptables -t nat -A POSTROUTING -o eth0 -j MASQUERADE
PostDown = iptables -D FORWARD -i %i -j ACCEPT; iptables -D FORWARD -o %i -j ACCEPT; iptables -t nat -D POSTROUTING -o eth0 -j MASQUERADE

To do this, I added the following to my JSON File
  "server": {
    "privateKey": "/****privateKey=",
    "publicKey": "/****publicKey=",
    "address": "10.8.0.1",
    "PostUp": "iptables -A FORWARD -i %i -j ACCEPT; iptables -A FORWARD -o %i -j ACCEPT; iptables -t nat -A POSTROUTING -o eth0 -j MASQUERADE",
    "PostDown": "iptables -D FORWARD -i %i -j ACCEPT; iptables -D FORWARD -o %i -j ACCEPT; iptables -t nat -D POSTROUTING -o eth0 -j MASQUERADE"
  },

However, upon reboot, I still don't see those two lines in wg0.conf. Is there something else I need to do in order to make that addition?

**Comment by Agilurso** [comment]
I came here to ask the same question.  PRE/POST Ups are usually performed in the WG0.conf file, however that file is overwritten upon reboot.  Where/How would we add PRE/Post UPs for local LAN accessibility with this implementation of Wireguard?

**Comment by WeeJeWel** [comment, marked as answer]
Since v5 you can overwrite PostUp with `WG_POST_UP` and PostDown with `WG_POST_DOWN`.
